libs/device/dialog/meta.json

```diff
@@ -0,0 +1,11 @@
+{
+    "width": 16,
+    "height": 16,
+    "blockIdentity": "images._image",
+    "creator": "image.ofBuffer",
+    "star": {
+        "namespace": "sprites.dialog",
+        "mimeType": "image/x-mkcd-f4"
+    },
+    "tags": "dialog"
```

Comment:
is this string displayed to the user? if so, what is the localization plan?

Reply:
right now it's just used internally for filtering, putting up the other pr now